val.py

```diff
@@ -430,10 +603,11 @@ def run(
         except Exception as e:
             LOGGER.info(f'pycocotools unable to run: {e}')
 
+
     # Return results
     model.float()  # for training
     if not training:
-        s = f"\n{len(list(save_dir.glob('labels/*.txt')))} labels saved to {save_dir / 'labels'}" if save_txt else ''
+        s = f"\n{len(list(save_dir.glob('labels/*.txt')))} labels in total saved to {save_dir / 'labels'}" if save_txt else ''
```

Comment:
what do you mean here?

Reply:
When reading the original comment it looks like the current run saved x amount of images, but it is in total for a specific folder, because it is doing: save_dir.glob() here. 

The comment confused me

Reply:
but this is saving labels right? im still confused how adding the 'total' word helps here?

Reply:
Oke, this better?

f"\nTotal {len(list(save_dir.glob('labels/*.txt')))} labels found in the folder {save_dir / 'labels'}"